docs/pages/api-reference/operators/window.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Window
+order: 0
+status: published
+---
+### Window
+
+<Divider>
+<LeftSection>
+Operator to do aggregation of events into windows based on their timestamps. Window operator must always be preceded by a [groupby](/api-reference/operators#groupby) operator.
+
+#### Parameters
+
+<Expandable title="type" type="Window">
+
+The type of window to group event on. Possible values are `"hopping"`, `"tumbling"`, `"session"`.
+
+`"hopping"`: Take in two parameters `"duration"` and `"stride"`, the `"duration"` parameters indicate how long the window is while `"gap"` indicate how far apart the window should be. Hopping window can overlap meaning one event can belong to two different windows. `"stride"` should be smaller than `"duration"`.
+
```

Comment:
You are specifying stide and duration but later using gap in the description ?

Can we change to something like -

A hopping window is defined by two key parameters: duration and stride.

< sublist >
duration: The length of the window, indicating the time span that each window covers.

stride (also referred to as gap in some contexts): The interval between the start of each window. This setting allows for the creation of overlapping windows, where a single event can be included in multiple windows. It's essential that the stride is less than the duration.